package.json

```diff
@@ -4,10 +4,13 @@
   "description": "Testing module for lumino node",
   "main": "src/index.js",
   "scripts": {
-    "test": "npx jest --runInBand",
+    "prepare": "npx husky install && scripts/validateResources",
+    "pretest": "scripts/validateResources",
+    "requirements": "scripts/validateResources",
+    "test": "scripts/runTests",
     "prettier": "npx prettier --write --ignore-unknown .",
     "prettier-check": "npx prettier --check .",
-    "prepare": "npx husky install"
+    "clean": "scripts/clean"
```

Comment:
same question here, when should i use this?

Reply:
The `clean` script it's just to cleanup your environment, remove dependencies folder and docker images, it's useful for testing purposes